tests/integration/statistics-components/project-information.test.ts

```diff
@@ -0,0 +1,38 @@
+import { expect, test, describe } from "vitest";
+import { render, screen } from "@testing-library/svelte";
+import ProjectInformation from "$lib/components/composites/statistics-components/ProjectInformation.svelte";
+import { waitForComponentLoading } from "../test-helper";
+import { loading } from "../../model-builder";
+
+describe("ProjectInformationComponent", () => {
```

Comment:
Until now, we only used the component name as test suite name, is there a reason why you treat it differently now?
```suggestion
describe("ProjectInformation", () => {
```

Reply:
Same in `stage-progress.test.ts`

Reply:
No, that wasn't intentional, I'm changing it so that it's consistent.